docs/application/dotnet/tutorials/certificates/getting-the-certificates.md

```diff
@@ -1,35 +1,34 @@
 # Getting the Certificates
 
-In order to install a Tizen application onto Samsung Tizen wearable devices, you must first register certificates containing the DUID (Device Unique Identifier) of the device.
-Two types of certificates are needed; the author and the distributor certificate.
-These certificates are used for signing and verifying the application.
+In order to install Tizen applications on Samsung Tizen wearable devices, you must first register certificates containing the Device Unique Identifier (DUID) of a device.
+There are two types of certificates, author certificate and distributor certificate. These certificates are used for signing and verifying the application.
 
-These certificates ensure that your signed application will run only on devices which you registered.
-The certification process is necessary not only for testing your apps on the actual devices but also for uploading your apps to Galaxy Apps.
+These certificates ensure that your signed application runs only on those devices that are registered by you. 
+The certification process is necessary not only for testing your apps on the actual devices but also for uploading them to Galaxy Store. 
 
-It guarantees that your application will not be distributed by someone else even if your application package is leaked.
-This document describes how to generate certificates from Samsung and how to maintain them also packaging the application.
-
-Contents:
+The certification process guarantees that your application will not be distributed by someone else even if your application package is leaked.
+The following pages describe how to generate certificates, how to maintain them, and how to package your application:
 
 - [Installing Certificates Extension](installing-certificate-extension.md)
 - [Creating Certificates](creating-certificates.md)
-- [Permit Device To Install Applications](permit-device-to-install-apps.md)
+- [Signing Application with Certificate](signing-your-application-with-a-certificate.md)
```

Comment:
Would you please check this link again?
I think the file name is changed to signing-your-application.md.

Reply:
Thank you for pointing out.
I will fix the link.